pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -541,7 +557,11 @@ var _ = Describe("Pod Network", func() {
 
 				api.NewDefaulter(runtime.GOARCH).SetObjectDefaults_Domain(domain)
 
-				bindMechanism, err := podnic.getPhase2Binding(vmi, &vmi.Spec.Domain.Devices.Interfaces[0], &vmi.Spec.Networks[0], domain, primaryPodInterfaceName)
+				podnic.vmi = vmi
+				podnic.iface = &vmi.Spec.Domain.Devices.Interfaces[0]
+				podnic.network = &vmi.Spec.Networks[0]
+				podnic.podInterfaceName = primaryPodInterfaceName
+				bindMechanism, err := podnic.getPhase2Binding(domain)
```

Comment:
Could you export this into a function ? It is repeated 15 times 0_o.

Reply:
It's slightly different sometimes, I will keep it like this, apart from the fact that it's not related to the PR.

Reply:
It's just hard on the eyes. This file change by > 150 lines and > 80 are because of this. 

Well, I'm OK with addressing it eventually.